inline-r/src/Foreign/R/Internal.hsc

```diff
@@ -0,0 +1,339 @@
+-- |
+-- Copyright: (C) 2013 Amgen, Inc.
+--
+-- Low-level bindings to core R datatypes and functions. Nearly all structures
```

Comment:
Redundant module docstring? Surely this module should have its own docstring, not the same one as `Foreign.R`?